examples/server/tests/unit/test_chat_completion.py

```diff
@@ -4,22 +4,26 @@
 
 server = ServerPreset.tinyllama2()
 
-
-@pytest.fixture(scope="module", autouse=True)
+@pytest.fixture(autouse=True)
```

Comment:
I'm not exceptionally good at `pytest` so maybe I'm missing something. Could you explain why `scope="module"` is removed here?

Reply:
scope=module was making the ServerProcess server instance shared between all the test in the module (file). Even though it's stopped in stop_server_after_each_test, it carried previous settings over to subsequent tests, spilling server flags over (became more important w/ jinja & chat_template params)

Reply:
Ok thanks for the explanation. Seem like module=scope is not what I wanted. I want the fixture to only affect single file, since the idea is that one test unit uses one model